src/Item.php

```diff
@@ -1203,11 +1221,11 @@ public function get_enclosures()
             // PLAYER
             if ($player_parent = $this->get_item_tags(\SimplePie\SimplePie::NAMESPACE_MEDIARSS, 'player')) {
                 if (isset($player_parent[0]['attribs']['']['url'])) {
-                    $player_parent = $this->sanitize($player_parent[0]['attribs']['']['url'], \SimplePie\SimplePie::CONSTRUCT_IRI);
+                    $player_parent = $this->sanitize($player_parent[0]['attribs']['']['url'], \SimplePie\SimplePie::CONSTRUCT_IRI, $this->get_base($player_parent[0]));
```

Comment:
Though then then if would fail my expectations here:

```php
        yield 'Test RSS 2.0 with channel link and relative enclosure link' => [
            <<<XML
            <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:media="http://search.yahoo.com/mrss/">
                <channel>
                    <link>http://example.net/tests/</link>
                    <item>
                        <link>http://item.example.net/tests/6/</link>
                        <media:content url="/images/6.jpg" medium="image"></media:content>
                    </item>
                </channel>
            </rss>
XML
            ,
            'http://item.example.net/images/6.jpg',
        ];
```

Reply:
Done

Reply:
Sorry, I meant all the get_base references in get_enclosures.

Reply:
Something like that? https://github.com/simplepie/simplepie/pull/861/commits/52f232fe36569bc58cf9bef55c81fdce5079784f